tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -2220,22 +2299,21 @@ public:
    /// defined columns returned by GetDefinedColumnNames().
    ///
    /// Please note that this is a convenience feature and the layout of the output can be subject
-   /// to change and should not be automatically parsed.
+   /// to change and should be parsed via RDFDescription methods.
    ///
    /// ### Example usage:
    /// ~~~{.cpp}
    /// RDataFrame df(10);
    /// auto df2 = df.Define("x", "1.f").Define("s", "\"myStr\"");
    /// // Describe the dataframe
-   /// std::cout << df2.Describe() << std::endl;
+   /// df2.Describe().Print()
+   /// df2.Describe().Print(/*shortFormat=*/true)
+   /// std::cout << df2.Describe().AsString() << std::endl;
+   /// std::cout << df2.Describe().AsString(/*shortFormat=*/true) << std::endl;
    /// ~~~
    ///
-   std::string Describe()
+   RDFDescription Describe()
```

Comment:
Why `RDFDescription` and not `RDescription` as in `RDisplay`? The namespace `ROOT::RDF` already qualifies it as an RDF thing no?

Reply:
Yes, I'm asking if there's a reason to name it `RDFDescription` and not `RDescription`.

Reply:
Yes, we might have other descriptions of things still under the `ROOT::RDF` namespace, and in Doxygen (or when searching the code in general) it's better to search for the `RDFDescription` class than for a generic `RDescription` class across all ROOT.

Reply:
mmh on second thought I don't know, Enric is right that we have some seriously generic names in the `ROOT::RDF` namespace, like `RDisplay` and `RInterface`.

I guess the question is just whether we'll ever have another description of something else, like a `RDF::RFilterDescription` :thinking:

Reply:
> I guess the question is just whether we'll ever have another description of something else, like a RDF::RFilterDescription

In that case, for our peace of mind, we can argue that `ROOT::RDF::RDescription` means the description of the RDF (the `RDF` namespace qualifies it as such), if the description is of an RDF filter, it is fair that it's called `ROOT::RDF:RFilterDescription`

Reply:
Peace of mind would be an argument for leaving the more specific name :smile:

Ivan, Enric, pick the one you like best, I have no preference.

Reply:
I think we should also think if later on, we will have other `RDF` "instances" which are used to "present" some specific part of RDF itself. As an example potentially `RDFMetaData`.
I believe answer should converge to yes, and hence I would go for `RDFDescription`. Moreover, I just saw that there is a `RDFTypeNameGetter` class, so keeping `RDF` seem consistent.

Remark, no matter which version we choose, very likely the other version is never going to be used (mainly for consistency and avoiding confusion - might be obvious but I want to make sure it is considered).